troubleshooting/tools/log-loss-test-framework/awslogs-benchmarking-scripts/fargate-task.json

```diff
@@ -0,0 +1,92 @@
+{
+    "family": "awslogs-benchmarking-fargate-${MAX_BUFFER_SIZE}-buffer-automated-250K-log",
+    "taskRoleArn": "arn:aws:iam::144718711470:role/ecs_task_role", 
+    "executionRoleArn": "arn:aws:iam::144718711470:role/ecsTaskExecutionRole",
+    "networkMode": "awsvpc",
+    "cpu": "1024",
+    "memory": "2048",
+    "containerDefinitions": [
+        {
+            "essential": false,
+            "image": "144718711470.dkr.ecr.us-west-2.amazonaws.com/log-loss-benchmark-logger:latest",
+            "name": "logger",
+            "logConfiguration": {
+                "logDriver": "awslogs",
+                "options": {
+                    "awslogs-group": "awslogs-benchmarking-output",
+                    "awslogs-region": "${REGION}",
+                    "awslogs-create-group": "true",
+                    "awslogs-stream-prefix": "pre-testing-bug",
+                    "mode": "non-blocking",
+                    "max-buffer-size": "${MAX_BUFFER_SIZE}"
+                }
+            },
+            "environment": [
+				{
+					"name": "SIZE_IN_KB",
+					"value": "${SIZE_IN_KB}"
+				},
+                {
+					"name": "TOTAL_SIZE_IN_MB",
+					"value": "${TOTAL_SIZE_IN_MB}"
+				},
+                {
+					"name": "THROUGHPUT_IN_KB",
+					"value": "${THROUGHPUT}"
+				},
+                {
+					"name": "CYCLE_TIME_IN_SECONDS",
+					"value": "1"
+				}
```

Comment:
Should we fix the indention  ??

Reply:
Also, why others are env variable but not this `"1"`

Reply:
The CYCLE_TIME_IN_SECONDS doesn't change in any of the tests, so I didn't add it as an env